modules/apiserversource-yaml.adoc

```diff
@@ -193,15 +220,25 @@ spec:
 
 .Verification steps
 
-You can verify that the Kubernetes events were sent into the Knative eventing system by looking at the message dumper function logs.
+You can verify that the Kubernetes events were sent to Knative by looking at the message dumper function logs.
+
+* View the message dumper function logs by entering the following commands:
 
-You can view the message dumper function logs by entering the following commands:
+[source,terminal]
 ----
 $ oc get pods
+----
+
++
```

Comment:
This `+` renders in the output. It can be removed when the two commands are separated into their own steps.